generator/generator_config.textproto

```diff
@@ -21,6 +21,22 @@ service {
   initial_copyright_year: "2021"
 }
 
+# Bigtable
+service {
+  service_proto_path: "google/bigtable/admin/v2/bigtable_instance_admin.proto"
+  product_path: "google/cloud/bigtable/admin"
+  initial_copyright_year: "2021"
+  emulator_endpoint_env_var: "BIGTABLE_INSTANCE_ADMIN_EMULATOR_HOST"
+}
+
+service {
+  service_proto_path: "google/bigtable/admin/v2/bigtable_table_admin.proto"
+  product_path: "google/cloud/bigtable/admin"
+  initial_copyright_year: "2021"
+  emulator_endpoint_env_var: "BIGTABLE_EMULATOR_HOST"
+  gen_async_rpcs: ["CheckConsistency"]
```

Comment:
Consider removing the "snapshot" APIs:

https://github.com/googleapis/googleapis/blob/bcfd386c224fded5f5c5be7a17bc5884a2a52f59/google/bigtable/admin/v2/bigtable_table_admin.proto#L70-L74

We do not support them in the current APIs, and have no plans to (these APIs never left "private preview").

Reply:
Ah, thanks for pointing this out.